src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -369,7 +369,12 @@ private int instances(AppDeploymentRequest request) {
 	}
 
 	private String routePath(AppDeploymentRequest request) {
-		return request.getDeploymentProperties().get(ROUTE_PATH_PROPERTY);
+		String routePath = request.getDeploymentProperties().get(ROUTE_PATH_PROPERTY);
```

Comment:
Not sure if we can still fail fast by having this validation where the app deployment happens compared to the validation at the place where the deployment was requested (ex: at Skipper).

Reply:
But, it definitely makes sense to have this type of validation at the deployer implementations.

Reply:
It should actually even be pushed down more into the cf java client lib....